G1ANT.Addon.Selenium/G1ANT.Addon.Selenium.Tests/SeleniumVersionTest.cs

```diff
@@ -0,0 +1,35 @@
+﻿using G1ANT.Tests;
+using NUnit.Framework;
+using System;
+using System.Net;
+using System.Reflection;
+using System.Text.RegularExpressions;
+
+namespace G1ANT.Addon.Selenium.Tests
+{
+    /// <summary>
+    /// Summary description for SeleniumVersionTest
+    /// </summary>
+    [TestFixture]
+    [TestCategory(TestCategories.Slow)]
+    public class SeleniumVersionTest
+    {
+
+        [Test]
+        public void ValidateSeleniumVersion()
+        {
+            int result = -1;
```

Comment:
### vaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaar

change to var everywhere

Reply:
no :(